### PR TITLE
don't use r-like extraction from greta

### PR DESCRIPTION
**Pull request by jjallaire**
@dfalbel pointed out here (https://github.com/rstudio/tensorflow/pull/199#issuecomment-342887212) that the feature complete version of r-like extraction doesn't perform nearly as well as the previous implementation. This PR uses the existing implementation (enhanced to optionally have a 1 rather than 0 basis) rather than the greta implementation.

@goldingn Let me know if this handles things correctly / as you imagined.

### CODE REVIEW

File: R/generics.R [hunk 1]
```diff
@@ -68,25 +68,12 @@ print.tensorflow.python.ops.variables.Variable <- print.tensorflow.python.framew
   check_zero_based(call)
 
   r_like_extract <- getOption("tensorflow.r_like_extract", TRUE)
```
Comment (goldingn) [inline R/generics.R]: It might be worth changing this back to `tensorflow.zero_based_extract` or similar.

Comment (goldingn) [reply R/generics.R]: otherwise, exactly what I had in mind & looks great!

**Comment by jjallaire** [comment]
Thanks for the feedback, I've changed the option to `tensorflow.one_based_extract = TRUE`. If you give me the thumbs up I'll go ahead and merge.

**Comment by goldingn** [comment]
:+1:

**Pull request merged**